rtdata/languages/default

```diff
@@ -137,7 +138,6 @@ FILEBROWSER_PARTIALPASTEPROFILE;Paste - partial
 FILEBROWSER_PASTEPROFILE;Paste
 FILEBROWSER_POPUPCANCELJOB;Cancel job
 FILEBROWSER_POPUPCOLORLABEL;Color label
-FILEBROWSER_POPUPCOLORLABEL0;Label: None
```

Comment:
I don't think this needs to be deleted. See https://github.com/Beep6581/RawTherapee/blob/2924128311d7fa5165a477b83a84c57870c3799b/rtgui/filebrowser.cc#L220-L222

Reply:
It's probably due to this bug: https://github.com/Beep6581/RawTherapee/pull/6503#issuecomment-1172974778
If there are two identical keys except one has an added `0` at the end, the one with the zero gets removed.

Reply:
I did not delete that :confused:

Reply:
Restored.